browser/BUILD.gn

```diff
@@ -112,7 +112,6 @@ source_set("browser_process") {
     "autoplay",
     "content_settings",
     "download",
-    "farbling",
     "net",
```

Comment:
brave_tab_helpers.cc uses farbling_tab_helper.h so if we move farbling_tab_helper to `//chrome/browser` then we need a `//chrome/browser` dep here

Reply:
Then, I wonder whether we can only move farbling_tab_helper.cc to //chrome/browser. If so, it looks like we don't need to add //chrome/browser here. WDYT?

Reply:
@bridiver I wonder if you think this makes sense.